src/runtime/pkg/containerd-shim-v2/errors.go

```diff
@@ -58,7 +58,8 @@ func isInvalidArgument(err error) bool {
 
 func isNotFound(err error) bool {
 	return err == vc.ErrNoSuchContainer || err == syscall.ENOENT ||
-		strings.Contains(err.Error(), "not found") || strings.Contains(err.Error(), "not exist")
+		strings.Contains(err.Error(), "not found") || strings.Contains(err.Error(), "not exist") ||
+		strings.Contains(err.Error(), "no such file or directory")
```

Comment:
Note: `isNotFound` is also called in the `StopContainer` case. However, I think that the added `no such file or directory` scenario also applies in that case.